requirements/runtime.txt

```diff
@@ -5,12 +5,12 @@ numpy
 Pillow<=6.2.2
 six
 terminaltables
-torch>=1.3
-torchvision
+torch==1.5.1
+torchvision==0.6.1
 pytorchcv==0.0.55
 tqdm
-onnx>=1.6.0
-onnxruntime>=1.1.2
+onnx==1.6.0
+onnxruntime==1.1.2
```

Comment:
Why is this?

Reply:
There were several times when code becomes non-working because of new version of pytorch (and probably onnx). I've decided to fix onnx stuff as well. Do you think it is a bad idea just to fix requirements?

Reply:
IMO it's more of a workaround than a solution, and it would be good to have more confidence in what have caused those issues. This being said, if latest versions of onnx and onnxruntime really make some troubles here, I don't mind fixing it this was.

Reply:
Why >= is actually needed? For most packages we use ==. With >= we always have a chance that something goes wrong and we can even don't know about it.

Reply:
And we won't have any advantage on the latest features, fixes and improvements at the same time. It can also make the whole project less user-friendly. I believe that possible regressions should be handled at a CI side. But as long as we don't have the ready to use infrastructure for that, then I don't mind disabling versions' flexibility.

Reply:
reverted flexibility, currently it does not create problems. Please "Approve" if you are ok with changes, as soon as CI is green.